pyccel/ast/core.py

```diff
@@ -2820,7 +2821,6 @@ def point(self, args, use_final_precision = False):
                     and (call_arg.rank == func_arg.rank) \
                     and call_arg.precision == func_arg.precision
 
-
```

Comment:
This file has not changed in a way that is useful for this PR. Please revert the changes to make the PR cleaner. (`git checkout master -- pyccel/ast/core.py`)